librz/util/subprocess.c

```diff
@@ -995,6 +1068,16 @@ RZ_API RzSubprocess *rz_subprocess_start_opt(RzSubprocessOpt *opt) {
 			rz_sys_pipe_close(stdin_pipe[1]);
 		}
 
+		if (opt->fork_mode == RZ_SUBPROCESS_FORKPTY) {
+			if (close(master_fd) == -1) {
+				perror("close");
+			}
+			if (rz_sys_login_tty(pty->slave_fd) == -1) {
+				perror("login_tty");
+				rz_sys_exit(-1, true);
+			}
```

Comment:
Is this really needed? If yes, why? Maybe a comment.

Reply:
![pty](https://user-images.githubusercontent.com/72350048/208911158-73cfe1c5-6dfc-4921-9d61-4ec1b4894d0c.jpg)

As labelled in the above chart, `login_tty` is responsible for connecting the child process' file descriptor to the slave file descriptor.

Reply:
Awesome 👍